packages/api/trpc/router.ts

```diff
@@ -1,6 +1,8 @@
 import { router, publicProcedure } from './base';
 import { sendEmail } from '../modules/email/procedures';
 import { authRouter } from '../modules/auth/procedures';
+import { usersRouter } from '../modules/users/procedures';
```

Comment:
nitpick: and then here import all the procedures for each module
```suggestion
import * as usersProcedures from '../modules/users/procedures';
```